Gemfile

```diff
@@ -21,10 +21,14 @@ gem 'jquery-rails'
 # bundle exec rake doc:rails generates the API under doc/api.
 gem 'sdoc', '~> 0.4.0', group: :doc
 
-
 gem 'passenger'
 gem 'rack-cors'
 
+gem 'faraday'
+gem 'faraday_middleware'
+
+gem 'figaro'
```

Comment:
Can we avoid using figaro?  I'd hate to add this to the list of packages we have to distribute just to use an environment variable.